setup.py

```diff
@@ -93,9 +102,12 @@ def run(self):
         _datafiles()
         _develop.run(self)
 
+get_patches()
```

Comment:
Oh, got it. I was viewing the reduced diff.

Would you mind integrating `get_patches()` into `_datafiles()`? That'll keep this process cleaner.